apps/transmitmedia.cpp

```diff
@@ -949,10 +949,10 @@ class UdpCommon
         {
             int ttl = stoi(attr.at("ttl"));
             int res = setsockopt(m_sock, IPPROTO_IP, IP_TTL, (const char*)&ttl, sizeof ttl);
-            if (res == -1)
+            if (transmit_verbose && res == -1)
                 cout << "WARNING: failed to set 'ttl' (IP_TTL) to " << ttl << endl;
```

Comment:
This `transmit_verbose` is now renamed to `Verbose::on`. And it's easier to simply change `cout` into `Verb()` and remove `<< endl`.